src/ng1/services.ts

```diff
@@ -158,6 +158,11 @@ function ng1UIRouter($locationProvider) {
 
   // Create a new instance of the Router when the ng1UIRouterProvider is initialized
   router = new UIRouter();
+  
+  // Apply ng1 `views` builder to the StateBuilder
+  router.stateRegistry.decorator("views", ng1ViewsBuilder);
+
+  router.viewService.viewConfigFactory('ng1', ng1ViewConfigFactory);
```

Comment:
Register the ng1 ViewConfig factory function (returns an `Ng1ViewConfig` object from an `Ng1ViewDeclaration`)